server/controller/model/agent_group_config_example.yaml

```diff
@@ -934,3 +934,9 @@ vtap_group_id: g-xxxxxx
   #################
   ## Note: Unreleased deepflow-agent features can be turned on by setting this switch.
   #feature-flags:
+
+  ###################
+  ## GuardInterval ##
+  ###################
+  ## Note: Time interval for the collector to detect its own state, unit: second.
+  #guard_interval: 60s
```

Comment:
放到 feature flag 前面。

另外参考其他的来写注释：
- 一行名称
- 一行说明 Default、Ranage 等
- 多行说明 Note

```
  ######################################
  ## Agent Running in Standalone Mode ##
  ######################################
  ## Data File Size
  ## Default: 200. Unit: MB.
  ## Note: When deepflow-agent runs in standalone mode, it will not be controlled by
  ##   deepflow-server, and the collected data will only be written to the local file.
```

Reply:
另外说明下 detect 哪些内容。

Reply:
特别是可能要说明一下代价。可以先给我贴一段中文的看看

Reply:
################
  ## Guard Interval ##
  ################
  ## agent 检测自身状态的时间间隔.
  ## Default: 60. Unit: s.
  ## Note:
  ## 采集器会检测的有：
  ## 1. 系统空闲内存
  ## 2. 采集器自身进程数
  ## 3. 采集器自身线程数
  ## 4. 采集器产生的日志文件大小及文件数量
  ## 由于采集器通过读取/proc路径下的文件获取以上信息，因此
  ## 会消耗资源，所以时间间隔越短，读取越频繁，占用cpu 会更多
  #guard-interval: 60s

Reply:
####################
  ## Guard Interval ##
  ####################
  ## agent 检测自身状态的时间间隔.
  ## Default: 60. Unit: s. Range: [xxx, xxx]
  ## Note: 采集器会检测的有：
  ##   1. 系统空闲内存
  ##   2. 通过读取 /proc 路径下的文件信息，获取采集器自身进程数
  ##   3. 通过读取 /proc 路径下的文件信息，获取采集器自身线程数
  ##   4. 采集器产生的日志文件大小及文件数量
  #guard-interval: 60s